config/locales/server.en.yml

```
@@ -2605,7 +2605,7 @@ en:
     disable_mailing_list_mode: "Disallow users from enabling mailing list mode (prevents any mailing list emails from being sent.)"
     default_email_previous_replies: "Include previous replies in emails by default."
 
-    default_emoji_reactions: "Default favorites emoji reactions. Add up to 5 emojis for quick reaction."
+    default_emoji_reactions: "Add up to 5 emojis shown by default in chat to allow quick reaction."
```

it's 3 emoji (I think it was 5 at the very beginning)

```suggestion
    default_emoji_reactions: "Add up to 3 emojis shown by default in chat to allow quick reaction."
```